code/modules/projectiles/ammunition/ballistic/shotgun.dm

```diff
@@ -143,3 +143,32 @@
 	reagents.add_reagent(/datum/reagent/toxin/acid/fluacid, 5) //this and the acid equal about 25ish burn, not counting the minute toxin damage dealt by their metabolism, this makes each dart about as lethal as a stechkin shot in upfront damage
 	reagents.add_reagent(/datum/reagent/toxin/acid, 5)
 	reagents.add_reagent(/datum/reagent/consumable/frostoil, 10) //tempgun slowdown goes both ways and adds to the burn
+
+//////////////////
+//    20 BORE   //
+//////////////////
+
+/obj/item/ammo_casing/shotgun_20bore
+	name = "20 bore shotgun slug"
+	desc = "A 20 gauge lead slug."
+	icon_state = "20slug"
+	caliber = "shotgun20bore"
+	projectile_type = /obj/item/projectile/bullet/shotgun_slug/twentybore_slug
+	custom_materials = list(/datum/material/iron=3000)
+
+/obj/item/ammo_casing/shotgun_20bore/buckshot
```

Comment:
You need to add the pellet count to this ammo type. Check buckshot. Otherwise it only shoots one pellet.

Reply:
I think originally this was a child of normal buckshot, so it'd inherit the code for firing 6 pellets and the variance. I'll make sure to double check this, thanks a ton.